paasta_tools/monitoring_tools.py

```diff
@@ -45,7 +45,8 @@ def monitoring_defaults(key):
                'https://paasta.readthedocs.io/en/latest/yelpsoa_configs.html#monitoring-yaml',
         'ticket': False,
         'project': None,
-        'realert_every': -1
+        'realert_every': -1,
+        'chronos_job_acceptable_delay': 15 * 60,  # 15 minutes
```

Comment:
I would rather this not be a first-class key on the monitoring dictionary, and instead by an input to a chronos config. (without a prefix of "chronos")